filebeat/etc/filebeat.yml

```diff
@@ -30,6 +30,14 @@ filebeat:
       # * stdin: Reads the standard in
       input_type: log
 
+      # Exclude lines. A list of regular expressions to match. It drops the lines that are
+      # matching any regular expression from the list. By default, no lines are dropped. 
+      # exclude_lines: ["^DBG"]
+
+      # Include lines. A list of regular expressions to match. It exports the lines that are
+      # matching any regular expression from the list. By default, all the lines are exported.
+      # include_lines: ["^ERR", "^WARN"]
```

Comment:
I just realised, you updated filebeat.yml instead of beat.yml. This is the auto generated file.